cmd/podman/containers/stop.go

```diff
@@ -107,6 +112,15 @@ func stop(cmd *cobra.Command, args []string) error {
 		args = append(args, id)
 	}
 
+	if len(filters) > 0 {
+		for _, f := range filters {
+			split := strings.SplitN(f, "=", 2)
+			if len(split) == 1 {
```

Comment:
FWIW, I try to stay away from `== 1` in tests like this and prefer `< 2`.  I think it's a tad bit more robust, YMMV.

Reply:
Yea, that makes sense.